src/app/shared/dovetail/dovetail-fetcher.ts

```diff
@@ -72,14 +79,6 @@ export class DovetailFetcher {
       if (request.getResponseHeader(CONTENT_TYPE) === APPLICATION_JSON) {
         if (request.readyState >= 4) {
           let response =  JSON.parse(request.responseText) as DovetailResponse;
```

Comment:
It made more sense to have the Dovetail/Adzerk remote fetchers just return the raw data, without attempting any parsing.  Any additional logic is handled by the DovetailSegment now.